lambdas/issuecredential/build.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
 			configurations.jackson,
 			configurations.sqs,
 			configurations.kms
+	testImplementation project(path: ':lambdas:drivingpermitcheck')
```

Comment:
Remove this -  the check lambda is not a test fixture for issue credential lambda.